common/src/main/java/org/schemarepo/config/Config.java

```diff
@@ -90,6 +105,9 @@
 
     DEFAULTS.setProperty(DEFAULT_SUBJECT_VALIDATORS,"");
 
+    // Validation defaults. The read strategy class is "Always Fail" unless configured.
+    DEFAULTS.setProperty(VALIDATION_READ_STRATEGY_CLASS, "org.schemarepo.validation.AlwaysFailValidationStrategy");
```

Comment:
Actually, the configuration still needs to be made currently. (The module being registered simply makes the class available to use as an implementation for the 'CanReadStrategy') I imagine it may be possible to inject default settings in the module and still allow them to be overridden in the config settings. I'm not sure how to do this.